app/controllers/readers_controller.rb

```diff
@@ -9,7 +9,7 @@ def update
     @reader = Reader.find_or_create_by(user: current_user,
                                        thread: @thread)
     @reader.touch
-    @anything_left = current_user.media_latest_comments.any? do |m|
+    @anything_left = current_user.subscribed_media_with_latest_comments_not_by_creator.any? do |m|
       (Reader.find_by(user: current_user, thread: m[:thread])
             &.updated_at || 1000.years.ago) < m[:latest_comment].created_at
     end
```

Comment:
In some later PR, the code duplication in `main_controller.rb` and `readers_controller.rb` should be removed, e.g. by writing a method for the last two lines.